_integrator-guide/jsonata-mapper-numbers.md

```diff
@@ -0,0 +1,148 @@
+---
+title: JSONata mapper | Working with numbers
+layout: article
+section: Data transformation
+category: data-transformation
+order: 2
+---
+
+The advanced mode of the [data mapping](mapping-data#using-advanced-mapping-mode)
+between integration components on {{site.data.tenant.name}} uses
+[JSONata](http://jsonata.org) expressions to transform incoming data into the
+necessary outgoing structure for the next component to consume.
+
+**In this article we will dive deeper into the methods of data transformations using
+the `numbers`.**
```

Comment:
I would tend to disagree with you on this subject. There are numerous cases where you need to do manipulation of incoming data to aggregate it into something useful for your next steps. The given example makes that obvious.

Reply:
I guess my argument is that I haven't seen the below use cases in practice but this could just be a matter of us interacting with different sets of users.